lib/api/controller/auth.js

```diff
@@ -536,6 +582,40 @@ class AuthController extends NativeController {
       request.context.token,
       request.input.args.expiresIn);
 
+    // Only if the support of Browser Cookie as Jwt is enabled
+    // otherwise we should send a normal response because
+    // even if the SDK / Browser can handle the cookie,
+    // Kuzzle would not be capable of doing anything with it
+    if (global.kuzzle.config.security.supportBrowserCookieAsJwt) {
+      const isCookieOnly = request.input.args.cookieOnly;
+
+      if (isCookieOnly) {
+        const resultContent = { 
+          _id: token.userId,
+          expiresAt: token.expiresAt,
+          ttl: token.ttl,  
+        };
+
+        request.setResult(resultContent,
```

Comment:
```suggestion
        request.setResult(null,
```

Since the first argument is not used by Kuzzle to construct the response (but rather the return of the controller action method), we tend to put `null` as the first argument here to avoid confusion. 

Also, this makes `resultContent` useless